programs/bpf_loader/src/syscalls/cpi.rs

```diff
@@ -842,12 +842,9 @@ fn check_authorized_program(
             && !(bpf_loader_upgradeable::is_upgrade_instruction(instruction_data)
```

Comment:
the clippy `allow` attached to this big if:

```
    #[allow(clippy::blocks_in_if_conditions)]
```

can be removed now. :)

Reply:
Done, thanks!